pom.xml

```diff
@@ -412,7 +412,7 @@
       <dependency>
         <groupId>com.github.cbioportal</groupId>
         <artifactId>session-service</artifactId>
-        <version>0.3.0</version>
```

Comment:
Any reason that we are not using latest version `0.4.0` at here?

Reply:
@dippindots don't remember. could you try to update and test it?